test/sqllogictest/pg_catalog_views.slt

```diff
@@ -15,12 +15,12 @@ CREATE VIEW test_view1 AS SELECT 1
 statement ok
 CREATE VIEW test_view2 AS SELECT 2
 
-query TTT colnames
+query TTTT colnames
 SELECT * FROM pg_catalog.pg_views WHERE viewname LIKE 'test_%'
 ----
-schemaname  viewname    viewowner
-public      test_view1  NULL
-public      test_view2  NULL
+schemaname  viewname    viewowner  definition
+public      test_view1  NULL       SELECT␠1
+public      test_view2  NULL       SELECT␠2
```

Comment:
The value stored in `definition` is actually slightly different in Postgres: They store an extra space at the beginning and a semicolon at the end, so the table would look like:

```
schemaname  viewname    viewowner  definition
public      test_view1  NULL        SELECT␠1;
public      test_view2  NULL        SELECT␠2;
```

We could easily use the same format, but I'm not sure whether we should. AFAICT, this exact format is not documented anywhere ([the docs say "view definition (a reconstructed SELECT query)"](https://www.postgresql.org/docs/8.0/view-pg-views.html)) and might change in future versions (at least the leading space, which kind of looks like a bug).

Adapting the Postgres format would also change the output of `pg_get_viewdef` (making it the same as Postgres'), and consequently the output of `\sv [view]`. For example:

```
CREATE OR REPLACE VIEW public.t_view AS
SELECT a, b FROM [u2 AS materialize.public.t]
```

vs.

```
CREATE OR REPLACE VIEW public.t_view AS
 SELECT a, b FROM [u2 AS materialize.public.t];
```

Reply:
Whitespace gets ignored in parsing so I wouldn't worry too much about it; if we find that someone expects there to be extra whitespace for whatever reason, we could always add it back in.

Reply:
Makes sense. Do you have an opinion about the semicolon?